core/src/main/scala/akka/persistence/cassandra/CassandraPluginConfig.scala

```diff
@@ -14,6 +14,10 @@ import akka.persistence.cassandra.session.CassandraSessionSettings
 
 case class StorePathPasswordConfig(path: String, password: String)
 
+object CassandraDispatchers {
+  val nameOfDefaultDispatcher = "cassandra-plugin-default-dispatcher"
```

Comment:
Why is this default constant needed, shouldn't the default come from reference.conf?

Reply:
looks like a left over, it's not used